pkg/model/printer_column.go

```diff
@@ -77,8 +77,13 @@ func sortFunction(sortByField string) func(a, b *PrinterColumn) bool {
 		return func(a, b *PrinterColumn) bool {
 			return a.JSONPath < b.JSONPath
 		}
+	case "index":
+		return func(a, b *PrinterColumn) bool {
+			return a.Index < b.Index
+		}
```

Comment:
Should this be the first case in the switch? Sorting by index should be preferred over any other?

Reply:
I don't how moving this case could change the default/preferred value. `sortByField` will have one of the values.

Reply:
Did you mean moving it to the default case? or maybe somehow set a default for `$resource.print.sortBy` ?

Reply:
Oh no I totally misread this method. Sorry